tests/GenerateScriptRegressionTests/GenerateScriptRegressionTestsFixture.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Xunit.Abstractions;
+
+namespace GenerateScriptRegressionTests;
+
+public class GenerateScriptRegressionTestsFixture
+{
+    public string WorkingDirectory { get; set; }
+
+    public GenerateScriptRegressionTestsFixture()
+    {
+        WorkingDirectory = Environment.CurrentDirectory.Substring(0, Environment.CurrentDirectory.IndexOf("source-build-reference-packages") + 31);
```

Comment:
I find the `WorkingDirectory` terminology here confusing.  To me the working directory for the `VerifyGenerateScript`  is the sandbox folder.  For other tests in the future it may be the `bin` directory.  To me this is really the `SBRPRoot` or `RepoRoot` directory.

Reply:
Changed to RepoRoot